subsys/net/lib/lwm2m_client_utils/lwm2m/location_assistance_obj.c

```diff
@@ -54,269 +54,14 @@ static int32_t pgps_start_gps_day;
 static int32_t pgps_start_gps_time_of_day;
 //Assistance data
 static char assist_buf[4096];
-
-static char rawData[3023] = {
```

Comment:
I think the commits could be cleaned up a bit, there's really no point introducing changes in a PR just to remove them in the next commit.

Reply:
Yeah, I agree. That is something that I have been working on and still there's work to do. At least in this file.